web/js/tasks.js

```diff
@@ -1193,7 +1193,7 @@ Ext.onReady(function(){
             // Default button for full-holiday task
             new Ext.Button({
                 id: 'fullHolidayTaskButton',
-                text: 'Full-Holiday task',
+                text: 'Full Vacation Day task',
```

Comment:
```suggestion
                text: 'Full vacation day task',
```
Suggesting small caps here for consistency with the other buttons.